client/etcd/config_test.go

```diff
@@ -66,6 +66,7 @@ func TestConfig(t *testing.T) {
 	require.Equal(t, 10*time.Second, cfg.SDConfig.InitTimeout)
 }
 
+// nolint: unparam
```

Comment:
Yea, it's because we always pass the same string for `contents`, so it suggests removing it as a parameter.